application/views.py

```diff
@@ -62,21 +64,23 @@ def save_existing_application(request, application_id):
             else:
                 f.is_draft = True
             f.save()
-            return HttpResponseRedirect(reverse("dashboard:application:index"))
+            return HttpResponseRedirect(
+                reverse("dashboard:application:all_applications", args=[user_type])
+            )
     else:
         form = HighSchoolApplicationForm()
     context = {"form": form, "application_id": application_id}
     return render(request, "application/index.html", context)
 
 
-def index(request):
+def all_applications(request, user_type):
     # TODO user_id will be replaced by sessions
     user_id = 1
     context = {"applications": HighSchoolApplication.objects.filter(user_id=user_id)}
     return render(request, "application/index.html", context)
 
 
-def detail(request, application_id):
+def detail(request, user_type, application_id):
     application = HighSchoolApplication.objects.get(pk=application_id)
     # TODO user_id will be replaced by sessions
```

Comment:
This should be done since sessions is already implemented and is available on develop

Reply:
refer to the comment above